package.json

```diff
@@ -26,11 +26,11 @@
 		"unit-test:watch": "jest --watch --coverage",
 		"integration-test": "node __tests__/integration/tester.js",
 		"integration-test:verbose": "node __tests__/integration/tester.js --verbose",
-		"integration-test:watch": "nodemon --watch __tests__/integration --exec npm run integration-test:verbose",
-		"start": "node dist/init.js",
+		"integration-test:start": "node dist/index.js --verbose -j ../__tests__/integration/mocks/uikit.json",
```

Comment:
Did we not have a test runner? That runs multiple tests?

Reply:
That happens here in code:
https://github.com/govau/furnace/blob/develop/__tests__/integration/tester.js#L151

Reply:
I think we need to talk about this one again :)

Reply:
I have removed the double test run from inside the tester file as we actually run 6 tests now